plugins/woocommerce/src/StoreApi/Routes/V1/Agentic/CheckoutSessions.php

```
@@ -0,0 +1,425 @@
+	 *
+	 * @param \WC_Order $order Order object.
+	 * @param \WC_Cart  $cart  Cart object.
+	 */
+	protected function update_order_from_cart( $order, $cart ) {
+		// Remove existing items
+		foreach ( $order->get_items() as $item_id => $item ) {
+			$order->remove_item( $item_id );
+		}
+
+		// Add cart items to order
+		foreach ( $cart->get_cart() as $cart_item_key => $cart_item ) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$cart_item_key'. (undefined)

(UnusedLocalVariable)